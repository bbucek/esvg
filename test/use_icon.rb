require 'esvg'
Esvg.icons({config_file: '_alias.yml', path: 'svg_icons'})
use = Esvg.svg_icon('chat-bubble')

def write_file(path, contents)
  FileUtils.mkdir_p(File.expand_path(File.dirname(path)))
  File.open(path, 'w') do |io|
    io.write(contents)
  end
end

write_file('build/use.html', use)
