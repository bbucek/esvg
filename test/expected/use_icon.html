<svg class="svg-icon icon-comments" viewBox="0 0 512 512" width="512" height="512"><use xlink:href="#icon-comments"/></svg>